diff --git a/src/platforms/rcore_ios.c b/src/platforms/rcore_ios.c
index c978d91f..72b36d83 100644
--- a/src/platforms/rcore_ios.c
+++ b/src/platforms/rcore_ios.c
@@ -55,0 +56 @@ extern void ios_destroy();
+extern void ios_resign_active();
@@ -755,0 +757 @@ static void SendGestureEvent(NSSet<UITouch*>* touches, int action) {
+    ios_resign_active();
diff --git a/src/rcore.c b/src/rcore.c
index f3a6b3fc..6bc57ce4 100644
--- a/src/rcore.c
+++ b/src/rcore.c
@@ -269,2 +269,2 @@ __declspec(dllimport) unsigned int __stdcall timeEndPeriod(unsigned int uPeriod)
-typedef struct { int x; int y; } Point;
-typedef struct { unsigned int width; unsigned int height; } Size;
+typedef struct { int x; int y; } CorePoint;
+typedef struct { unsigned int width; unsigned int height; } CoreSize;
@@ -284,10 +284,10 @@ typedef struct CoreData {
-        Point position;                     // Window position (required on fullscreen toggle)
-        Point previousPosition;             // Window previous position (required on borderless windowed toggle)
-        Size display;                       // Display width and height (monitor, device-screen, LCD, ...)
-        Size screen;                        // Screen width and height (used render area)
-        Size previousScreen;                // Screen previous width and height (required on borderless windowed toggle)
-        Size currentFbo;                    // Current render width and height (depends on active fbo)
-        Size render;                        // Framebuffer width and height (render area, including black bars if required)
-        Point renderOffset;                 // Offset from render area (must be divided by 2)
-        Size screenMin;                     // Screen minimum width and height (for resizable window)
-        Size screenMax;                     // Screen maximum width and height (for resizable window)
+        CorePoint position;                 // Window position (required on fullscreen toggle)
+        CorePoint previousPosition;         // Window previous position (required on borderless windowed toggle)
+        CoreSize display;                   // Display width and height (monitor, device-screen, LCD, ...)
+        CoreSize screen;                    // Screen width and height (used render area)
+        CoreSize previousScreen;            // Screen previous width and height (required on borderless windowed toggle)
+        CoreSize currentFbo;                // Current render width and height (depends on active fbo)
+        CoreSize render;                    // Framebuffer width and height (render area, including black bars if required)
+        CorePoint renderOffset;             // Offset from render area (must be divided by 2)
+        CoreSize screenMin;                 // Screen minimum width and height (for resizable window)
+        CoreSize screenMax;                 // Screen maximum width and height (for resizable window)
diff --git a/src/rlgl.h b/src/rlgl.h
index 9c270e5c..918b59e6 100644
--- a/src/rlgl.h
+++ b/src/rlgl.h
@@ -877 +876,0 @@ RLAPI void rlLoadDrawQuad(void);     // Load and draw a quad
-    #include <libGLESv2/GLES/glext.h>
